projects/sbb-esta/angular-business/src/lib/contextmenu/contextmenu.module.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { ButtonModule, DropdownModule } from '@sbb-esta/angular-public';
```

Comment:
Use relative imports for now. I have opened an issue to solve this for the whole project: #147